diff --git a/node_modules/react-native-maps/android/build.gradle b/node_modules/react-native-maps/android/build.gradle
index 88b8991..57265d1 100644
--- a/node_modules/react-native-maps/android/build.gradle
+++ b/node_modules/react-native-maps/android/build.gradle
@@ -46,7 +46,7 @@ dependencies {
   implementation 'com.facebook.react:react-native:+'
   implementation "com.google.android.gms:play-services-base:${safeExtGet('playServicesVersion', '18.1.0')}"
   implementation "com.google.android.gms:play-services-maps:${safeExtGet('playServicesVersion', '18.0.2')}"
-  implementation "com.google.android.gms:play-services-location:20.0.0"
+  implementation "com.google.android.gms:play-services-location:21.0.0"
   implementation 'com.google.maps.android:android-maps-utils:3.4.0'
   implementation "androidx.work:work-runtime:2.7.1"
 }
diff --git a/node_modules/react-native-maps/android/build/.transforms/d4d92e8e9a525e4f71665ea43d17fe35/transformed/out/META-INF/com/android/build/gradle/aar-metadata.properties b/node_modules/react-native-maps/android/build/.transforms/d4d92e8e9a525e4f71665ea43d17fe35/transformed/out/META-INF/com/android/build/gradle/aar-metadata.properties
new file mode 100644
index 0000000..8c9c699
--- /dev/null
+++ b/node_modules/react-native-maps/android/build/.transforms/d4d92e8e9a525e4f71665ea43d17fe35/transformed/out/META-INF/com/android/build/gradle/aar-metadata.properties
@@ -0,0 +1,4 @@
+aarFormatVersion=1.0
+aarMetadataVersion=1.0
+minCompileSdk=1
+minAndroidGradlePluginVersion=1.0.0
